commands/getuser.js

```diff
@@ -34,7 +34,7 @@ async function execute(state, message, args) {
     // Check command sender is Administrator or has the exec role or higher
     const guildmember = guild.member(message.author);
 
-    if (!utils.is_exec_or_higher(guild.roles, guildmember.roles)
+    if (!utils.is_exec_or_higher(guild.roles, guildmember.roles)) {
```

Comment:
This won't work? The and is not going to interact properly

Reply:
Thanks for the catch.